product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+  - **Memory** (Gauge chart) &mdash; Displays the percentage of memory utilized by the node hosting the PostgreSQL cluster.
+  - **CPU** (Gauge chart) &mdash; Displays the percentage of CPU utilized for the PostgreSQL cluster.
+  - **Storage** (Gauge chart) &mdash; Displays the percentage of the storage volume utilized by the PostgreSQL cluster.
+  - **Storage** (Donut chart) &mdash; Displays the units of the storage volume utilized and available for the PostgreSQL cluster.
+  - **Active connections** (Text value) &mdash; Displays the current number of connections between the client applications and the PostgreSQL cluster.
+  - **Disk throughput** (Text value) &mdash; Displays the amount of data transferred to and from the disk per second.
+  - **Transactions per second** (Text value) &mdash; Displays the total number of transactions executing within a second for the PostgreSQL cluster.
+  - **Database size** (Text value) &mdash; Displays the amount of storage volume utilized by the database.
+
+## Historical charts
+
+It displays the historical data of last 15 minutes by default. To view the historical data of a particular time range, customize the time range using a time range picker. These charts display key metrics in single line and bar or multi line and bar form. It provides a concise snapshot of the information such as:
+  - **Memory** (Line chart) &mdash; Displays the historical trend of memory usage percentage over a time period.
+  - **CPU** (Bar chart) &mdash; Displays the historical trend of CPU usage percentage over a time period.
+  - ** Network activity** (Bar chart) &mdash; Displays the historical data transfer to and from the network card per second, over a time period.
+  - **Disk IOPS** (Multi-line chart) &mdash; Displays the historical trends in the number of reds, writes, and total operations on the disk per second, over a time period.
+  - **Transaction per second** (Multi-line chart) &mdash; Displays the historical trends in the number of transactions per second, over a time period.
+
+
+## Features for both types of charts
+
+All these charts has some tools and features that helps getting more information on the metrics or about the chart. The [time range picker](#time-range-picker) helps in viewing the data on these charts for a specific time range interval. The [information tootip](#information-tooltip) helps to view the information for that particular chart. The [charts error state](#charts-error-state) helps to find out the error and provides the option to edit the configurations and fix the error.
+
+### Time range picker
+
+On `Monitoring and Logging` tab, to view the data of a particular time range configure the time range using:
+- the `Last X` combo box
```

Comment:
Just commenting about nitpicks only because the doc is user-facing 😄. There are some typos over this doc and the combobox name changes slightly across the pages. We can solve them at the end in a final "aestetic" review.  I can help with that. Thank you very much for all this work so far 😄.

Reply:
sure. Thanks!